sentinel_s3/converter.py

```diff
@@ -36,15 +36,37 @@ def epsg_code(geojson):
     return None
 
 
-def convert_coordinates(coords, origin, wgs84):
-    """ Convert coordinates from one crs to another """
+def test_wrap_coordinates(coords, origin, wgs84):
+    """ Test whether coordinates wrap around the antimeridian in wgs84 """
+    lon_under_minus_170 = False
+    lon_over_plus_170 = False
     if isinstance(coords, list) or isinstance(coords, tuple):
         try:
             if isinstance(coords[0], list) or isinstance(coords[0], tuple):
-                return [convert_coordinates(list(c), origin, wgs84) for c in coords]
+                return [test_wrap_coordinates(list(c), origin, wgs84) for c in coords]
```

Comment:
I think this still applies test_wrap_coordinates on the coordinates multiple times because the coordinates are wrapped into multiple lists